go/cmd/dolt/commands/status.go

```diff
@@ -86,15 +87,15 @@ func (cmd StatusCmd) Exec(ctx context.Context, commandStr string, args []string,
 		handleStatusVErr(err)
 	}
 
-	err = PrintStatus(ctx, dEnv, staged, notStaged, as)
+	err = PrintStatus(ctx, dEnv, staged, notStaged, apr.Contains(cli.ShowIgnoredFlag), as)
 	if err != nil {
 		return handleStatusVErr(err)
 	}
 	return 0
 }
 
 // TODO: working docs in conflict param not used here
```

Comment:
I know you didn't do this one, but could you pull out this line while you're in here? It doesn't seem to make any sense any more. 🤷‍♂️ 

```suggestion
```